package.json

```
@@ -185,8 +186,11 @@
     "serve:dist": "gulp serve:dist",
     "test:auto": "gulp test:auto",
     "buildNoReg": "gulp buildNoReg",
-    "lint": "gulp lint",
-    "lint:fix": "gulp lint:fix",
-    "compile": "gulp compile"
+    "lint": "gulp lint && npm run prettier",
+    "lint:fix": "gulp lint:fix && npm run prettier:fix",
+    "compile": "gulp compile",
+    "prettier": "prettier --check \"**/*.{ts,html,scss,json}\"",
+    "prettier:list": "prettier --list-different \"**/*.{ts,html,scss,json}\"",
+    "prettier:fix": "prettier --write \"**/*.{ts,html,scss,json}\""
```

I think we also have `css` and `js` files
```suggestion
    "prettier": "prettier --check \"**/*.{js,ts,html,css,scss,json}\"",
    "prettier:list": "prettier --list-different \"**/*.{js,ts,html,css,scss,json}\"",
    "prettier:fix": "prettier --write \"**/*.{ts,html,cs,scss,json}\""
```